packages/terra-paginator/src/ControlledPaginator.jsx

```diff
@@ -113,6 +119,7 @@ class Paginator extends React.Component {
   }
 
   defaultPaginator() {
+    const { intl } = this.props;
```

Comment:
Could this be moved into the object destructuring two lines below this one?

Reply:
Resolved https://github.com/cerner/terra-core/pull/2485/commits/d961db300072dd0a36b415437ab98ac82efaf35f